### PR TITLE
Dev/sol/termination test ii

### PR DESCRIPTION
**Pull request by SoloJacobs**
Adds three commits to dev/sol/termination_test i

### CODE REVIEW

**Review by jherbel — APPROVED** [review]

File: v2/robotmk/tests/timeout/lib/create_child.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import subprocess
+
+
+def spawn() -> None:
+    with subprocess.Popen(["python", "-c", "import time; time.sleep(10)"]) as p:
```
Comment (jherbel) [inline v2/robotmk/tests/timeout/lib/create_child.py:5]: We can directly use `time.sleep` without a subprocess

File: v2/robotmk/tests/test_scheduler.rs [hunk 2]
```diff
@@ -17,6 +17,7 @@ use tokio::{process::Command, time::timeout};
 use walkdir::WalkDir;
 
 #[tokio::test]
+#[ignore]
```
Comment (jherbel) [inline v2/robotmk/tests/test_scheduler.rs:20]: This currently leads to the test also being skipped by our system tests

**Review by jherbel — CHANGES_REQUESTED** [review]

File: v2/robotmk/tests/timeout/lib/create_child.py (hunk 1 above)
Comment (SoloJacobs) [reply v2/robotmk/tests/timeout/lib/create_child.py:5]: Done

File: v2/robotmk/tests/test_scheduler.rs (hunk 2 above)
Comment (SoloJacobs) [reply v2/robotmk/tests/test_scheduler.rs:20]: Done:
https://github.com/elabit/robotmk/actions/runs/7195151926/job/19597218491?pr=454

**Review by jherbel — APPROVED** [review]

**Pull request merged**